doc/releases/release-notes-3.7.rst

```diff
@@ -846,6 +853,14 @@ Drivers and Sensors
 
   * The update channels function is now optional and can be left unimplemented.
 
+  * The ``in-gpios`` and ``output-pin`` properties of the respective ``worldsemi,ws2812-gpio.yaml``
+    and ``worldsemi,ws2812-rpi_pico-pio`` devicetree bindings have been renamed to ``gpios``.
```

Comment:
```suggestion
  * The ``in-gpios`` and ``output-pin`` properties of the respective :dtcompatible:`worldsemi,ws2812-gpio`
    and :dtcompatible:`worldsemi,ws2812-rpi_pico-pio` devicetree bindings have been renamed to ``gpios``.
```

Reply:
Done.

Reply:
Without the ".yaml" in the first one :)

Reply:
Sorry I am doing too many things at the same time...